install/kubernetes/cilium/templates/_helpers.tpl

```diff
@@ -5,6 +5,14 @@ Create chart name and version as used by the chart label.
 {{- printf "%s-%s" .Chart.Name .Chart.Version | replace "+" "_" | trunc 63 | trimSuffix "-" }}
 {{- end }}
 
+{{/*
+Return fqdn cilium image
```

Comment:
I don't understand this comment, can you please take a look and see if you can rephrase it?